plugins/modules/network_compliance_workflow_manager.py

```diff
@@ -0,0 +1,1390 @@
+- API to sync device configuration on device(s).
+version_added: "6.13.0"
+extends_documentation_fragment:
+  - cisco.dnac.workflow_manager_params
+author: Rugvedi Kapse (@rukapse)
+        Madhan Sankaranarayanan (@madhansansel)
+options:
+  config_verify:
+    description: Set to True to verify the Cisco Catalyst Center config after applying the playbook config.
+    type: bool
+    default: False
+  state:
+    description: State of Cisco Catalyst Center after module completion.
+    type: str
+    choices: [ merged ]
+    default: merged
+  config:
+    description: List of device details for running a compliance check or synchronizing device configuration.
+    type: list
+    elements: dict
+    required: True
+    suboptions:
+      ip_address_list:
+        description: List of IP addresses of devices to run a compliance check on or synchronize device configurations.
+                     Either "ip_address_list" or "site_name" is required for module to execute.
+                     If both "site_name" and "ip_address_list" are provided, operations are performed on devices that are present in both the
+                     "ip_address_list" and the specified site.
+                     (e.g. ["204.1.2.2", "204.1.2.5", "204.1.2.4"])
+        type: list
+        elements: str
+      site_name:
+        description: When "site_name" is specified, the module executes the operation on all the devices located within the specified site.
+                     This is a string value that should represent the complete hierarchical path of the site.
+                     Either "site_name" or "ip_address_list" is required for module to execute.
+                     If both "site_name" and "ip_address_list" are provided, operations are performed on devices that are present in both the
+                     "ip_address_list" and the specified site.
+                     (e.g. "Global/USA/San Francisco/Building_2/floor_1")
+        type: str
+      run_compliance:
+        description: Determines if a full compliance check should be triggered on the devices specified in the "ip_address_list" and/or "site_name".
+                     if it is True then compliance will be triggered for all categories.
+                     If it is False then compliance will be not be triggered for all categories.
+        type: bool
+        default: True
+      run_compliance_categories:
+        description: Specifying compliance categories allows you to trigger compliance checks only for the mentioned categories.
+                     Category can have one or more values from among the options "INTENT", "RUNNING_CONFIG", "IMAGE", "PSIRT", "EOX", "NETWORK_SETTINGS".
+                     Category "INTENT" is mapped to compliance types "NETWORK_SETTINGS", "NETWORK_PROFILE", "WORKFLOW", "FABRIC", "APPLICATION_VISIBILITY".
+                     (e.g. ["INTENT", "RUNNING_CONFIG", "IMAGE", "PSIRT", "EOX", "NETWORK_SETTINGS"])
```

Comment:
Can we add one statement saying 
if "run_compliance" is set to "False", then compliance will be not be triggered for the given categories.